src/main/java/org/jitsi/jicofo/jigasi/TranscriberManager.java

```diff
@@ -181,14 +182,15 @@ private TranscriptionStatusExtension getTranscriptionStatus(Presence p)
      * Method which is able to invite the transcriber by dialing Jigasi
      * @param preferredRegions a list of preferred regions.
      */
-    private void startTranscribing(Collection<String> preferredRegions)
+    private void startTranscribing(@NotNull Collection<String> preferredRegions)
     {
         if (active)
         {
             return;
         }
 
-        selectTranscriber(2, null, preferredRegions);
+        // We need a modifiable list for the "exclude" parameter.
```

Comment:
Is this comment still the case?  `selectTranscriber` looks like it takes an immutable list.

Reply:
Yes. This is java, so it's not immutable. Line 251 adds to it...

What I did in this commit was replace this with a @NotNull annotation:
```
if (exclude == null)
{
    exclude = new ArrayList<>();
}
```

Reply:
Oops.  Ok gotcha.